src/decorators/JSONField.ts

```diff
@@ -0,0 +1,27 @@
+// tslint:disable-next-line: no-var-requires
+const GraphQLJSONObject = require('graphql-type-json').GraphQLJSONObject;
```

Comment:
Does this not have an option that supports `import` syntax?

Reply:
fixed